src/core_plugins/metrics/public/components/vis_editor.js

```diff
@@ -110,7 +110,8 @@ class VisEditor extends Component {
             dirty={this.state.dirty}
             autoApply={this.state.autoApply}
             model={model}
-            visData={this.props.visData}
+            savedObj={this.props.savedObj}
```

Comment:
This should not be defined when not running in editor mode, since in that case it would be `ReactVisType` that renders that method and it only passes in the `vis` class.